gateway/ln-gateway/src/lib.rs

```diff
@@ -900,7 +911,7 @@ impl Gateway {
                 let balance =
                     bitcoin::Amount::from_sat(client.value().get_balance().await.msats / 1000);
                 let fees = wallet_module
-                    .get_withdraw_fees(address.clone(), balance)
+                    .get_withdraw_fees(address.as_unchecked().clone(), balance)
```

Comment:
Shouldn't we modify methods to take "checked" address instead? We should generally use `Checked` all over the code, except in deserialization / taking input from the user, where we sould make sure to validate them.

Reply:
Good point! Fixed.